packages/xsnap/lib/console-shim.js

```diff
@@ -1,16 +1,13 @@
 /* global globalThis */
+
+// We use setQuote() below to break the cycle
+// where SES requires console and console is
+// implemented using assert.quote from SES.
+let quote = v => (typeof v === 'string' ? v : '[?]');
+
 function tryPrint(...args) {
-  try {
-    // eslint-disable-next-line
-    print(...args.map(arg => typeof arg === 'symbol' ? arg.toString() : arg));
-  } catch (err) {
-    // eslint-disable-next-line
-    print('cannot print:', err.message);
-    args.forEach((a, i) => {
-      // eslint-disable-next-line
-      print(` ${i}:`, a.toString ? a.toString() : '<no .toString>', typeof a);
-    });
-  }
+  // eslint-disable-next-line
+  print(...args.map(v => typeof v === 'string' ? v : quote(v)));
```

Comment:
The Node.js console joins console arguments with a single space `' '`.

Reply:
so does XS print: ``await worker.evaluate(`print('sep?', 1, 2, 3);`);`` prints
```
sep? 1 2 3
```

Reply:
Perfect.